hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java

```diff
@@ -53,13 +43,26 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
 import java.util.Random;
 import java.util.Set;
 import java.util.stream.Collectors;
-
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.generic.IndexedRecord;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.apache.parquet.avro.AvroParquetReader;
+import org.apache.parquet.avro.AvroReadSupport;
+import org.apache.parquet.hadoop.ParquetReader;
+import org.apache.spark.Partitioner;
 import org.apache.spark.api.java.JavaSparkContext;
-import scala.Option;
+import org.apache.spark.api.java.function.Function2;
+import org.apache.spark.api.java.function.PairFlatMapFunction;
 import scala.Tuple2;
+import scala.Option;
```

Comment:
Yea, it is being used here (below). I have re-organized imports, so it has moved from top to bottom. I can undo that if you want me to.

https://github.com/kaushikd49/hoodie/blob/ca5232ab386a5a0d05c56a1724304641e7dad224/hoodie-client/src/main/java/com/uber/hoodie/table/HoodieCopyOnWriteTable.java#L363